dsa/README.md

```diff
@@ -0,0 +1,52 @@
+# dsa
```

Comment:
Can you use one of our existing README.mds as a template? We try to keep them fairly consistent across the project:

https://github.com/RustCrypto/signatures/blob/master/ecdsa/README.md

Thanks!

Reply:
I hope the about section in the new README is good enough. I wasn't exactly sure what to write there